Palace/AppInfrastructure/TPPAppTheme.swift

```diff
@@ -25,7 +25,7 @@
       return colorFromHex(hex(theme))
```

Comment:
Do you think we can use available system colors for iOS 13+ here for matching values? they are already optimised for dark mode, and change automatically when iOS changes dark mode value.